modules/services/sxhkd.nix

```
@@ -73,6 +73,7 @@ in
     ];
 
     xsession.initExtra = ''
+      systemctl --user stop sxhkd.scope
```

```suggestion
      systemctl --user stop sxhkd.scope || true
```

If an error occurs, is the script evaluation aborted?

---

No, just throw an error to journal. I added check if scope is active.